tests/test_headers.py

```diff
@@ -14,10 +14,21 @@
 
 from astro_metadata_translator import merge_headers, fix_header, HscTranslator
 from astro_metadata_translator.tests import read_test_file
+from astro_metadata_translator import DecamTranslator
 
 TESTDIR = os.path.abspath(os.path.dirname(__file__))
 
 
+class NotDecamTranslator(DecamTranslator):
+    """This is a DECam translator with override list of header corrections."""
+    name = None
+
+    @classmethod
+    def fix_header(cls, header):
+        header["DTSITE"] = "hi"
```

Comment:
Just curious: what is `DTSITE` supposed to refer to?

Reply:
It's a header I picked at random from the test header that was being used. It has a value of "ct" in the header and I was checking that this callback triggered and changed it to "hi".